js/layouts/tab-container/tab-container.js

```diff
@@ -326,18 +330,33 @@ $(function()
 	         * @return {string} The [data-tab-id] of the iIdx-th tab (zero based). Can return undefined if it has not [data-tab-id] attribute
 	         * @private
 	         */
-	        _getTabIdFromTabIndex(iIdx) {
+	        _getTabIdFromTabIndex: function(iIdx) {
 		        return this.element.children(this.js_selectors.tabs_list).children(this.js_selectors.tab_header).eq(iIdx).attr('data-tab-id');
 	        },
 	        /**
 	         * @param sId {string} The [data-tab-id] of the tab
 	         * @return {number} The index (zero based) of the tab. If no matching tab, 0 will be returned.
 	         * @private
 	         */
-	        _getTabIndexFromTabId(sId) {
+	        _getTabIndexFromTabId: function(sId) {
 		        const oTabElem = this.element.children(this.js_selectors.tabs_list).children(this.js_selectors.tab_header+'[data-tab-id="'+sId+'"]');
 
 		        return oTabElem.length === 0 ? 0 : oTabElem.prevAll().length;
-	        }
+	        },
+            _getTabElementFromTabIndex: function(iIdx) {
+                return this.element.children(this.js_selectors.tabs_list).children(this.js_selectors.tab_header).eq(iIdx);
+            },
+            disableTab: function(iIdx){
+               let tabsWidget = this.GetTabsWidget();
+               tabsWidget.disable(iIdx);
+               let tabElement = this._getTabElementFromTabIndex(iIdx);
```

Comment:
We rather use `const` as the variable itself isn't supposed to be rewritten.

```suggestion
               const tabsWidget = this.GetTabsWidget();
               tabsWidget.disable(iIdx);
               const tabElement = this._getTabElementFromTabIndex(iIdx);
```